src/packages/excalidraw/CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ Please add the latest change on the top under the correct section.
 
 ### Features
 
+- [`restore(data, localAppState, localElements)`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#restore) and [`restoreElements(elements, localElements)`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#restoreElements) now take `localElements` argument which will be used to ensure existing elements' versions are used and incremented. This fixes an issue where importing the same file would resolve to elements with older versions, potentially causing issues when reconciling [#3797](https://github.com/excalidraw/excalidraw/pull/3797).
```

Comment:
technically a breaking change

Reply:
it breaks APIs coz you need to supply the `localElements`

Reply:
```suggestion
- [`restore(data, localAppState, localElements)`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#restore) and [`restoreElements(elements, localElements)`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#restoreElements) now take `localElements` argument which will be used to ensure existing elements' versions are used and incremented. This fixes an issue where importing the same file would resolve to elements with older versions, potentially causing issues when reconciling [#3797](https://github.com/excalidraw/excalidraw/pull/3797).

### BREAKING CHANGE
- If you were using [`restore`](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#restore) or [`restoreElements`]((https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/README.md#restoreElements) you will need to pass the last argument `localElements` as its a required attribute. You can pass `null` if not applicable.
```

Ohh yes I forgot we are making it mandatory, so lets mention it as a breaking change